webapp/src/modules/analytics/track.ts

```diff
@@ -246,8 +248,9 @@ track<ClaimAssetSuccessAction>(
 track<AcceptRentalListingSuccessAction>(
   ACCEPT_RENTAL_LISTING_SUCCESS,
   'Rent Land',
-  ({ payload: { periodIndexChosen, rental } }) => ({
+  ({ payload: { periodIndexChosen, rental, nft } }) => ({
     nftId: rental.nftId,
+    assetType: isParcel(nft) ? 'Parcel' : 'Estate',
```

Comment:
What if we use the `NFTCategory` type to name the asset types?
```suggestion
    assetType: isParcel(nft) ? NFTCategory.PARCEL : NFTCategory.ESTATE,
```